src/extension.ts

```diff
@@ -978,15 +982,12 @@ export async function activate(context: vscode.ExtensionContext): Promise<any> {
       ) {
         vscode.commands.executeCommand("editor.foldAll");
         openedClasses.push(editor.document.uri.toString());
-        workspaceState.update("openedClasses", openedClasses);
       }
     }),
     vscode.workspace.onDidCloseTextDocument((doc: vscode.TextDocument) => {
```

Comment:
Suggest you use a `const` for `doc.uri.toString()` in place of the two occurrences below.